index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Tumaini-Maganiko portfolio</title>
+</head>
+<body>
+    <header class="toolbar">
+        <div id="my-logo">
+            <span>My Logo</span>
+        </div>
+        <div class="menu-bar">
+            <img src="/images/Union.png" alt="">
+        </div>
+    </header>
+    <section class="headline-section">
+        <h1>Hey There. <br>I'm Tumaini.</h1>
+        <h2>I'm a Software Developer</h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+        <div class="social-media">
+            <button><img src="/images/github.png" alt="github"></button>
+            <button><img src="/images/linkedin.png" alt="linked in"/></button>
+            <button><img src="/images/anjelles.png" alt="anjelles"></button>
+            <button><img src="/images/twitter.png" alt="twitter"></button>
+            <button><img src="/images/medium.png" alt="medium icon"></button>
+        </div>
```

Comment:
Since this is a list, kindly group them by using the ul tag, `li` per each icon, and each icon in a link (a tag) instead of the image (img tag). This is good for semantic reasons as well as help you connect the icons to your social networks by using `href`. 😉